dev-resources/Documents/elrr_feature_branches.txt

```diff
@@ -0,0 +1,389 @@
+https://github.com/US-ELRR/elrrexternalservices - polls with an LRS to get xapi data
+https://github.com/US-ELRR/elrrdatasync - Grabs xAPI data via externalservices and dumps it onto a kafka topic
+https://github.com/US-ELRR/elrraggregator - aggregates all of the statements from the topic and writes them to the database accordingly
+https://github.com/US-ELRR/elrrservices - API layer to CRUD data in the database has context menu
+https://github.com/US-ELRR/elrrdockercompose
+https://github.com/phleven/elrr_documentation
+
+
+
+Create local feature branch dev-svc-update-local-fixes-and-docs-1 for elrrservices
+ 1. Open terminal
+      a. cd C:\Users\phleven\git\elrrservices
+      b. git switch svc-update
+      c. git switch -c dev-svc-update-local-fixes-and-docs-1
```

Comment:
again, you need your docs to be more general. This is something you would keep for yourself but you should not document basic git usage in the projects.